README.md

```diff
@@ -40,8 +40,26 @@ This is a portable library so you can use it anywhere.
 This library should be available on NuGet for easy consumption by your projects.
 You may also copy and paste the P/Invoke signatures you need directly into your projects if desired. 
 
+Packages:
+
+Library      | Package name     | NuGet       | Description
+-------------|------------------|-------------|-------------
+advapi32.dll |`PInvoke.AdvApi32`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.AdvApi32.svg)](https://www.nuget.org/packages/PInvoke.AdvApi32)|Windows Advanced Services
+bcrypt.dll   |`PInvoke.BCrypt`  | [![NuGet](https://img.shields.io/nuget/v/PInvoke.BCrypt.svg)](https://www.nuget.org/packages/PInvoke.BCrypt)|[Windows Cryptography API: Next Generation][CNG]
+gdi32.dll    |`PInvoke.Gdi32`   | [![NuGet](https://img.shields.io/nuget/v/PInvoke.Gdi32.svg)](https://www.nuget.org/packages/PInvoke.Gdi32)|[Windows Graphics Device Interface][Gdi]
+hid.dll      |`PInvoke.Hid`     | [![NuGet](https://img.shields.io/nuget/v/PInvoke.Hid.svg)](https://www.nuget.org/packages/PInvoke.Hid)|[Windows Human Interface Devices][Hid]
+kernel32.dll |`PInvoke.Kernel32`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.Kernel32.svg)](https://www.nuget.org/packages/PInvoke.Kernel32)|Windows Kernel API
+ncrypt.dll   |`PInvoke.NCrypt`  | [![NuGet](https://img.shields.io/nuget/v/PInvoke.NCrypt.svg)](https://www.nuget.org/packages/PInvoke.NCrypt)|[Windows Cryptography API: Next Generation][CNG]
+setupapi.dll |`PInvoke.SetupApi`| [![NuGet](https://img.shields.io/nuget/v/PInvoke.SetupApi.svg)](https://www.nuget.org/packages/PInvoke.SetupApi)|[Windows application programming interface][SetupApi]
```

Comment:
GitHub hides my last comment due to your resorting, but it was that your description column for setupapi should be "Windows *setup* API".
In particular, you were missing "setup", and I suggest use the "API" acronym since you use it elsewhere.